enhancements/api-review/add-ImagePolicy-to-config.openshift.io-v1alpha1.md

```diff
@@ -0,0 +1,336 @@
+3. The cluster administrator can get the merged cluster scope policies from `status` of ImagePolicy instances. 
+4. The cluster administrator can delete the signature verification configuration by deleting its ImagePolicy instances. 
+
+**application administrator** is a human user responsible for
+adding signature verification configurations for specific namespace.
+
+1. The application administrator creates ImagePolicy yaml file for namespace that is not `namespace: opesnhift-config`
+2. The application administrator creates ImagePolicy instance using `oc create -f imagepolicy.yaml`.
+3. The application administrator can get the merged cluster policy and policy for certain namespace from `status` of ImagePolicy instances. 
+4. The application administrator can delete the signature verification configuration by deleting its ImagePolicy instances. 
+
+### API Extensions
+
+#### Type definitions
+
+```go
+type ImagePolicySpec struct {
+	// images holds images/repositories to be verified
+	Images []Image `json:"images"`
+	// policy defines the verification policy
+	Policy Policy `json:"policy"`
+}
+
+// Image defines the list of images assigned to a policy.
+// For more information about the format, see the document
+// about the location field:
+// https://github.com/containers/image/blob/main/
+// docs/containers-policy.json.5.md#docker
+type Image struct {
+	Hostname string `json:"hostname,omitempty"`
+}
+
+type Policy struct {
+	// OIDCIssuer contains the expected OIDC issuer.
+	// Requires SubjectEmail to be set and KeyData to be empty.
+	// Example: "https://expected.OIDC.issuer/"
+	OIDCIssuer string `json:"oidcIssuer,omitempty"`
+
+	// SubjectEmail holds the email address of the subject.
+	// Example: "expected-signing-user@example.com"
+	// Requires OIDCIssuer to be set and KeyData to be empty.
+	SubjectEmail string `json:"subjectEmail,omitempty"`
+
+	// KeyData contains inline base64 data of the public key.
+	// Can be empty if the image got signed keyless.
+	// Requires OIDCIssuer and SubjectEmail to be empty.
+	KeyData string `json:"keyData,omitempty"`
+
+	// SignedIdentity specifies what image identity the signature
+	// claims about the image.
+	SignedIdentity Identity `json:"signedIdentity,omitempty"`
+
+	// FulcioCAData contains inline base64 data for the fulcio
+	// CA certificate.
+	// Defaults to the base64 encoded contents of:
+	// https://raw.githubusercontent.com/sigstore/root-signing/main/
+	// targets/fulcio_v1.crt.pem
+	// QUESTION: Can we omit it in favor of the upstream default?
+	FulcioCAData string `json:"fulcioCAData,omitempty"`
+
+	// RekorKeyData contains inline base64 data of the rekor public key.
+	// Defaults to the base64 encoded contents of:
+	// https://raw.githubusercontent.com/sigstore/root-signing/main/
+	// targets/rekor.pub
+	// QUESTION: Can we omit it in favor of the upstream default?
+	RekorKeyData string `json:"rekorKeyData,omitempty"`
```

Comment:
Let's drop it, we just use the defaults and can add the fields in a later API iteration.